tests/performance/Dockerfile

```diff
@@ -1,6 +1,9 @@
-FROM locustio/locust:1.4.4
+FROM locustio/locust:1.5.3
 
+COPY ./requirements /build/requirements
```

Comment:
I think hyou forgot to commit requirements folder?
Got
```
=> ERROR [2/3] COPY ./requirements /build/requirements  
```

Reply:
Or probably forgot to remove the line ... since you list the reqs below

Reply:
no I forgot to remove that line. thanks good catch!